radio/services/mail.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from aiohttp import ClientSession, BasicAuth
+from config.settings import (
+    MAIL_GUN_API_KEY,
+    MAIL_GUN_API_URL,
+    INFO_EMAIL,
+    FEEDBACK_SUBJECT
+)
+
+server_logger = logging.getLogger('aiohttp.server')
+
+
+async def send_mail(sender, body):
+    """
+    Args:
+        sender: email sender
+        body: text body of email that in going to be sent
+
+    """
+    try:
+        with ClientSession() as client:
+            await client.post(
+                MAIL_GUN_API_URL,
+                auth=BasicAuth("api", MAIL_GUN_API_KEY),
```

Comment:
Is MAIL_GUN_API_KEY = None correct arg?
looks like what we should have here logging error message if it's not defined.